src/pages/trade-options/trade-options.page.tsx

```diff
@@ -56,28 +67,49 @@ export default function TradeOptionsPage (): ReactElement {
             <section id="positions-section">
                 <div className="table-box">
                     <div className="table-wrapper">
-                        <div className="title">Positions</div>
+                        <div className="positions-title">Positions</div>
                         <table className="data-table">
                             <thead>
                                 <tr>
                                     <th>Expiry Date</th>
                                     <th>Strike Price</th>
                                     <th>Liquidity</th>
-                                    <th>Your Obligations ({currency.toUpperCase()})</th>
-                                    <th>Your Options ({currency.toUpperCase()})</th>
-                                    <th>Premium</th>
-                                    <th>Potential Performance</th>
+                                    <th>Positions</th>
+                                    <th>Performance</th>
                                 </tr>
                             </thead>
                             <tbody>
                                 <tr>
-                                    <td>{Math.floor(Math.random() * 10000)}</td>
-                                    <td>{Math.floor(Math.random() * 4)}</td>
-                                    <td>{Math.floor(Math.random() * 10000)}$</td>
-                                    <td>{Math.floor(Math.random() * 3)/100}</td>
-                                    <td>{Math.floor(Math.random() * 5)/100}</td>
-                                    <td>{Math.floor(Math.random() * 1000)}$</td>
-                                    <td>{Math.floor(Math.random() * 1000)}$</td>
+                                    <td>{new Date(1603598437527).toDateString().slice(4,15)}</td>
```

Comment:
The positions table should have the same width as the option tables above. Make sure that e.g. the colun for Liquidity and Positions has the same width as the tables above otherwise it looks inconsistent.

![image](https://user-images.githubusercontent.com/14966470/93196205-1009a500-f785-11ea-986b-0cc40258d50a.png)